### ISSUE TITLE
Better generator integration/example

### ISSUE THREAD

**Issue by drewbo** [developmentseed/label-maker#15]
Creating a `data.npz` file may result in too large a file for some projects. At that point it may be better to read the training data in with a generator (example: [Keras flow_from_directory](https://keras.io/preprocessing/image/)). The easiest way to do this now would be to skip the final step (`label-maker package`) and read directly from the `labels.npz` and the `tiles` directory. We should create either (1) an example showing how to do this or (2) additional scripts which perform this for the user depending upon configuration (or a combination of the two).

**Comment by wronk** [comment]
I vote for starting with (1) since it's the simplest and we avoid rewriting Keras's [ImageDataGenerator](https://keras.io/preprocessing/image/) functionality. If we want to show examples for other platforms (like MXNet or straight tensorflow) we can do (2), but then it might be worth separating that work into 2 issues.
